README.md

```diff
@@ -249,6 +240,168 @@ class UserController extends Controller {
 }
 ```
 
+## If your API doesn't follow our conventions
+Now it's time to introduce you some annotations helping configuring own routes. These class annotations must only be used for entities. The following entity is the one created in the chapter before, but with the additional ```DataSource```annotations.
```

Comment:
introduce you to some .., which help you to configure your own routes. Be sure to use them only with `Doctrine` entities. To demonstrate their capabilities, let's customize some parts of the previous chapter with the `DataSource` annotation: